controller/src/pid.cpp

```diff
@@ -116,16 +121,33 @@ void pid_execute(const VentParams &params, SensorReadings *readings) {
     break;
   }
 
+  //@TODO: The following are raw accesses that need to be updated to reference
+  // the new modules because currently they do not have the benefits of things
+  // like calibration, etc. that the module methods do. I didn't do this because
+  // I have no clue what the desired units are for the control system.
```

Comment:
My naive guess to your answer is that the units of the input to the control system seems to be "a number in the range [0, 255]".  So I think basically if we have a minimum/maximum pressure in kPa, we should be able to map that to [0, 255] and then call pid?

Reply:
I'd prefer not to take a naive approach without understanding what the model actually wants. Just because it was there before doesn't mean it was correct!

Reply:
Or relevant to what we're trying to do for alpha